_docs/operations/policies.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: guide
+title: StorageOS Docs - Policies
+anchor: operations
+module: operations/policies
+---
+
+# Policies
+
+Policies are a way to grant access to StorageOS namespaces. Policies can be
+configured at the group or user level so access can be controlled granularly.
+
```

Comment:
Nowhere says why a policy is used or helpful. Do we have a concept page for this?

I'd add a small note in the 1 paragraph mentioning that 

"the Administrator can restrict what namespaces each user can see to ensure privacy and isolation of responsibilities"

Reply:
I think that's more of a job for the concepts policy page. I made to to clearly spell out that users only have access to namespaces that they are mapped to with policies, except for the default namespace.